### PR TITLE
Require Chef 12.14 and remove yum dep

### PR DESCRIPTION
**Pull request by tas50**
Remove some old Chef 11 compatibility code in the metadata. Require a mildly current Chef 12 release and yank the yum dep. Remove duplicate apt deps. Cleanup docs for cookbook deps.

### CODE REVIEW

**Comment by sc7565** [comment]
@jlruizmlg  can we get this merged?

File: wazuh/metadata.rb [hunk 1]
```diff
@@ -29,6 +25,6 @@
 depends 'yum'
```
Comment (devsibwarra) [inline wazuh/metadata.rb]: `yum` dep pulled from the README, but still in metadata

File: wazuh_filebeat/metadata.rb [hunk 2]
```diff
@@ -6,7 +6,6 @@
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
 version          '0.1.1'
 
-
```
Comment (devsibwarra) [inline wazuh_filebeat/metadata.rb:9]: `wazuh_filebeat` is missing these lines.  Was this intentional or should these be included here?
```
issues_url 'https://github.com/wazuh/wazuh-chef/issues'
source_url 'https://github.com/wazuh/wazuh-chef'
chef_version '>= 12.14'
```

**Comment by tas50** [comment]
@devsibwarra should be good to go now thanks to @curiositycasualty

**Comment by rshad** [comment]
Hi @tas50, @curiositycasualty,

Thank you for contributing to Wazuh; Wazuh really gets improved by the community proposals and contributions.

This PR's changes are considered outdated due to the huge amount of changes and refactoring we added to the repository of `wazuh-chef`, so we will close it.

Kind regards,

Rshad

**Pull request closed without merging**